projects/driver_controls/inc/event_arbiter.h

```diff
@@ -16,12 +16,16 @@
 // Typedef for the FSM arbitration functions
 typedef bool (*EventArbiterCheck)(const Event *e);
 
-// Initializes the event arbiter to the default state
-StatusCode event_arbiter_init(void);
+typedef void (*EventArbiterCANOutput)(uint8_t device_id, uint8_t device_state, uint16_t data);
+
+// Initializes the event arbiter to the default state with a given output function
+StatusCode event_arbiter_init(EventArbiterCANOutput output);
```

Comment:
Don't explicitly make this CAN - the idea is that these modules can be completely independent of CAN.

Reply:
Done